api/src/pcapi/core/offers/api.py

```diff
@@ -948,7 +948,7 @@ def update_stock_id_at_providers(venue: Venue, old_siret: str) -> None:
 def get_expense_domains(offer: Offer) -> list[ExpenseDomain]:
     domains = {ExpenseDomain.ALL.value}
 
-    for deposit_type in deposit_conf.SPECIFIC_CAPS:
+    for deposit_type in deposit_conf.SPECIFIC_CAPS.items():
```

Comment:
Le code itérait sur un dictionnaire, donc sur ses clés. Maintenant ça itère sur des listes de tuples `(clé, valeur)`, je ne pense pas que ça va fonctionner.

(Je ne vois pas très bien de quoi pylint se plaignait ici. Peut-être qu'il n'aimait pas lister les clés pour ensuite accéder aux valeurs dans le corps de la boucle for ? En tout cas, accéder à `items()` n'est probablement pas la solution.)

Reply:
ça me disait `src/pcapi/core/offers/api.py:953: [C0206(consider-using-dict-items), get_expense_domains] Consider iterating with .items()`